content/SCALE/SCALETutorials/Apps/InstallNextCloudMedia.md

```diff
@@ -34,34 +38,37 @@ In this procedure you:
+To create the Nextcloud datasets, go to **Datasets**, select the dataset you want to use as the parent dataset, then click **Add Dataset** to [add a dataset]({{< relref "DatasetsScale.md" >}}). In this example, we create the Nextcloud datasets under the root parent dataset **tank**.
 
-To create a new pool, go to **Storage** and click **Create Pool** to [add a new pool]({{< relref "CreatePoolScale.md" >}}).
+Enter **nextcloud** in **Name**, make any other setting changes you want to make, and click **Save**.
 
-To add under an existing dataset, click the <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i>&nbsp; for the dataset where you want to add the NextCloud datasets, and then select **[Add Dataset]({{< relref "DatasetsSCALE.md" >}})**. 
-In our Nextcloud example we use pool *tank*, parent dataset *apps** and then created the *nextcloud* dataset. 
+Next, select the **nextcloud** dataset, click **Add Dataset** to add the **data** child dataset. 
+Enter **data** in **Name**, make any other setting changes you want to make for the dataset, and click **Save**. 
 
-Next, select the **nextcloud** dataset, click <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i>&nbsp; and select **Add Dataset** to add a child dataset. Enter **data** in **Name** and click **Save**. 
-Repeat this step three more times to add the three child datasets to the **nextcloud** dataset, one named **db**, the next **dbbackup**, and then finally **opt**.
+Repeat this three more times to add the other three child datasets to the **nextcloud** non-root parent dataset. 
+Add one named **db**, the next **dbbackup**, and then finally **opt**.
 
-When finished you should have the *nextcloud* parent dataset with four child datasets under it. Our example paths are:
-* */mnt/tank/apps/nextcloud/data*
-* */mnt/tank/apps/nextcloud/db*
-* */mnt/tank/apps/nextcloud/dbbackup*
-* */mnt/tank/apps/nextcloud/opt*
+When finished you should have the **nextcloud** parent dataset with four child datasets under it. Our example paths are:
+* */mnt/tank/nextcloud/data*
+* */mnt/tank/nextcloud/db*
+* */mnt/tank/nextcloud/dbbackup*
+* */mnt/tank/nextcloud/opt*
 
-![AppsAddNextCloudStorage](/images/SCALE/22.02/AppsAddNextCloudStorage.png "Add Nextcloud Storage")
+![AppsAddNextcloudDatasets](/images/SCALE/22.12/AppsAddNextcloudDatasets.png "Add Nextcloud Storage")
 
 ### Set Up the ffmpg Binary
 
+To set the ffmp binary, include `sudo` in the command and enter the admin password. 
 Go to **System > Shell** and enter these six commands:
 
 ```
-cd /mnt/tank/apps/nextcloud/opt
+cd /mnt/tank/nextcloud/opt
 
-wget https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-amd64-static.tar.xz
+sudo wget https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-amd64-static.tar.xz
```

Comment:
I don't have much experience with nextcloud stuff. But this seems like a somewhat inappropriate thing to have in official docs. If this is a feature we need in nextcloud then I feel like it should be a feature request to have backend do this in an automated (and correct) way. Having users type things in like this is just a recipe for fat-fingering input and breaking things.

Reply:
@stavros-k can we manage this in the app itself please with an additional question in nextcloud app as a install/update hook perhaps ( safely checking first if this already is setup or not ) ?

Pointing out this to users to do manually is really a recipe for disaster and bug reports where they might have missed some step or done something differently resulting in different outcome or breaking some other part.

Reply:
I've added Kris as a reviewer as this article follows instructions he provided last year. I'll make whatever changes we need made. FYI... this content has been out there for at least 6 months already but that doesn't mean it needs to remain in our docs.

Reply:
Agreed we should probably have this scripted in somehow. Nextcloud for reasons doesn't include ffmpeg, but it really is a almost deal-breaker type functionality to not have it available in the image.

Reply:
Removed the section on the ffmeg binary as directed.

Reply:
Maybe while the issue is still in people's minds, @micjohnson777 can you copy the original script into a new NAS ticket in Jira and link it to the docs one?